wd_demo.au3

```diff
@@ -398,7 +400,11 @@ Func SetupGecko()
 	_WD_Option('DriverParams', '--log trace')
 	_WD_Option('Port', 4444)
 
-	Local $sDesiredCapabilities = '{"capabilities": {"alwaysMatch": {"browserName": "firefox", "acceptInsecureCerts":true}}}'
+	_WD_CapabilitiesStartup()
+	_WD_CapabilitiesAdd('alwaysMatch', 'firefox')
+	_WD_CapabilitiesAdd('w3c', True)
+	_WD_CapabilitiesAdd('excludeSwitches', 'enable-automation')
```

Comment:
Not sure that this is even valid for Firefox. Shouldn't it be `acceptInsecureCerts` like in the line you are replacing? Also, I suggest that you somehow leave the original desiredCapabilities string (probably just a comment) so that the user can know what you intend to generate with the _WD_Capabilities* calls.

Reply:
sure, done